index.bs

```diff
@@ -967,6 +970,39 @@ A string or dictionary identifying a cryptographic algorithm and optionally a se
 defined in [[!WebCryptoAPI]].
 
 
+## Algorithm to Relax the Same-Origin Restriction ## {#algo-relax-same-origin-restriction}
+
+In general, the same-origin policy prevents web applications from interacting
+within user agents, unless they have the same origin. However, there are many
+specific facets and exceptions to this overall policy.
```

Comment:
I think if you want to keep this, it's best to put it this way:

> In general, the same-origin policy prevents pages from different origins from interacting with each other.  However, for payments we want to allow subdomains to create payment requests that look like they come from the main domain.

or so.  That is, explain what problem you're really trying to solve.  The general "there are some exceptions" verbiage mostly confuses people (including me) as to where we're heading with this.